### DISCUSSION TITLE
When will de CAM recordings be sync-ed?

### DISCUSSION THREAD

**Discussion by rgroothuis** [marcone/teslausb#782] [Q&A] [answered]
I was wondering when the CAM recordings will be synced with the external folder? Is this once a day at a specific time? Is this when the Raspberry makes a WIFI connection again (after being away)? Or do you need to trigger this manually via the web-site interface? Where can I find more info on this?

**Comment by marcone** [comment, marked as answer]
> Is this when the Raspberry makes a WIFI connection again (after being away)?

Yes. More specifically, when it can connect to the archive after having been unable to connect to it.

> Where can I find more info on this?

It's alluded to in the README, but perhaps not as clear as it could be.

![image](https://github.com/marcone/teslausb/assets/48169102/1bea032b-26d7-4da8-8dd0-2dede1ceee9b)

**Reply by rgroothuis** [reply]
thanks for the feedback. it is clear now.
